### PR TITLE
[Merged by Bors] - Update testnet configs, change on-disk format

### PR DESCRIPTION
**Pull request by paulhauner**
## Issue Addressed

- Related to #1691

## Proposed Changes

- Add `DEPOSIT_CHAIN_ID` and `DEPOSIT_NETWORK_ID` to `config.yaml`.
    - Pass the `DEPOSIT_NETWORK_ID` to the `eth1::Service`.
- Remove the unused `MAX_EPOCHS_PER_CROSSLINK` from the `altona` and `medalla` configs (see [spec commit](https://github.com/ethereum/eth2.0-specs/commit/2befe90032702b747e17e2f7531a41a4440d1342#diff-efb845ac2ebd4aafbc23df40f47ce25699255064e99d36d0406d0a14ca7953ec)).
- Change from compressing the whole testnet directory, to only compressing the genesis state file. This is the only file we need to compress and *not* compressing the others makes them work nicely with git.
    - We can modify the boot nodes, configs, etc. without incurring an eternal binary-blob cost on our git history.
    - This change is backwards compatible (i.e., non-breaking).

## Additional Info

NA

### CODE REVIEW

**Comment by paulhauner** [comment]
Hey @pawanjay176, would you mind giving this a look over please? I know you're familiar with it :)

File: consensus/types/src/chain_spec.rs [hunk 1]
```diff
@@ -108,6 +108,8 @@ pub struct ChainSpec {
      */
     pub eth1_follow_distance: u64,
     pub seconds_per_eth1_block: u64,
+    pub deposit_chain_id: u64,
```
Comment (pawanjay176) [inline consensus/types/src/chain_spec.rs:111]: We are currently checking only `network_id` in the eth1 crate to make sure beacon node is connected to the correct eth1 chain. I think we should check `chain_id` as well to ensure that we aren't reading any bogus logs from an eth1 chain with a different chain id.

File: common/eth2_config/src/lib.rs [hunk 2]
```diff
@@ -3,6 +3,18 @@ use std::env;
 use std::path::PathBuf;
 use types::ChainSpec;
 
+// A macro is used to define this constant so it can be used with `include_bytes!`.
```
Comment (pawanjay176) [inline common/eth2_config/src/lib.rs:6]: cool trick!

**Review by pawanjay176 — APPROVED** [review]
LGTM! just a minor nit and raising an unrelated issue.

File: common/eth2_testnet_config/.gitignore [hunk 3]
```diff
@@ -1,4 +1,5 @@
 testnet*/*
 schlesi-*
 witti-*
-altona*
+/altona*
```
[the remark this answers is not in the document]
Comment (paulhauner) [reply common/eth2_testnet_config/.gitignore:4]: I wanted to try and avoid everyone suddenly having dirty working trees when we merged it and maybe accidentally adding things with `git add -A`. Perhaps we can remove after a bit of time has passed.

File: consensus/types/src/chain_spec.rs (hunk 1 above)
Comment (paulhauner) [reply consensus/types/src/chain_spec.rs:111]: Good call, I made an issue here: https://github.com/sigp/lighthouse/issues/1815

**Comment by paulhauner** [comment]
Thanks @pawanjay176!

bors r+

**Comment by paulhauner** [comment]
bors retry

**Pull request closed without merging**